### ISSUE TITLE
Create PowerPoint for final presentation. 

### ISSUE THREAD

**Issue by ghost** [GatechVIP/no-code#49]
Create powerpoint for final presentation (include previous designs, and the results of user testing; show how the design changed, incorporating changes suggested by user testing)

**Comment by ghost** [comment]
For good luck:
![tumblr_lxa8dsfv6D1r7yvmwo1_400](https://f.cloud.github.com/assets/3238771/395499/a2224a00-a800-11e2-89fc-e374e0244105.gif)

**Comment by penghou620** [comment]
Does everyone should have his/her own PowerPoint?
—

**Comment by pongad** [comment]
No, the entire team should have one power point. Each member of the front-end team should be contributing to one (either student prototype or instructor prototype). The backend team will be contributing to two: the instructor prototype we helped with and the Great Hilary Experiment.

**Comment by penghou620** [comment]
How does the backend team split up the job?
—

**Comment by pongad** [comment]
That is a beautiful question. @Webs961 @sichinumi?
